src/main/riddl/Organization/organization.riddl

```diff
@@ -20,16 +20,16 @@ context OrganizationContext is {
             on other { then error "Only EstablishOrganization command is allowed in the default state" }
         }
 
-        record DraftState is {
-            info: OrganizationContext.Info,
+        record DraftTenantState is {
+            info: OrganizationContext.OptionalInfo,
             parent: OrganizationId?,
             members: MemberId+,
             owners: MemberId+,
             contacts: Contact?,
```

Comment:
Both this and the other one you noted are desired by James to be on top level. This was established in conversations a while ago. We don't want them in Info for various reasons, mostly to do with importance of use and so inappropriateness of putting them in Info.

In Draft state they are Optional so that we can check if they are there when `Activate` command is sent